client/dashboard/task-list/index.js

```diff
@@ -74,15 +75,32 @@ class TaskDashboard extends Component {
 		} ).filter( task => task.visible );
 	}
 
+	getPluginsInformation() {
+		const { isJetpackConnected } = this.props;
+		const plugins = [ 'jetpack', 'woocommerce-services' ];
+		const { activePlugins, installedPlugins } = getSetting( 'onboarding', {} );
+		const matchedInstalledPlugins = intersection( installedPlugins, plugins );
+		const matchedActivePlugins = intersection( activePlugins, plugins );
+		return {
+			wcs_installed: matchedInstalledPlugins.includes( 'woocommerce-services' ),
```

Comment:
Instead of using `matchedInstalledPlugins` could we simplify this by just using `installedPlugins.includes( 'woocommerce-services' )`?

Same question for the next 3 lines.

Reply:
Thank you Joshua, you are absolutely right! I will change it now.